.gitignore

```diff
@@ -1,14 +1,5 @@
 # Files and directories created by pub
-.buildlog
+.dart_tool
 .packages
-.project
 .pub
-**/build
-**/packages
-
-# Directory created by dartdoc
-doc/api/
```

Comment:
why remove this?

Reply:
packages dir is no longer created
We don't generate docs in this package